packages/react-app/src/hooks/useRoles.ts

```diff
@@ -11,16 +10,13 @@ export const useRoles = (): Role[] => {
 	 * Note, we currently only support roles in bankless, and need to establish a more complete
 	 * Auth method for multitenancy
    */
-	const { data: session } = useSession({ required: false });
-	const { data: roles, error } = useSWR<Role[], unknown>(
-		session
-			? ['/api/auth/roles', session.accessToken]
-			: null,
+	const { data, error } = useSWR<{ roles: Role[] }, unknown>(
+		'/api/auth/roles',
```

Comment:
We do not need a composite session key in the SWR due to cache logic in the service layer